### ISSUE TITLE
Why does IOP UHD require square pixels?

### ISSUE THREAD

**Issue by sandersaares** [Dash-Industry-Forum/DASH-IF-IOP#294]
I noticed that our UHD chapter says one must use square pixels. Is there a particular reason for this? This constraint was unexpected to me and the rationale is not obvious, so some clarification might be useful for confused readers such as me.

**Comment by haudiobe** [comment]
non-square pixel are not in common use anymore today. Why would you do this? UHD is consistent in using 16:9 and square pixel.

**Comment by sandersaares** [comment]
Does it not work with non-square pixels? I am asking to understand the reason we require square pixels.

If square pixels are required, this creates a dependency between the input resolution and the representation resolution. If they are not required, there is no dependency and you can use a fixed set of coded resolutions regardless of input aspect ratio, leading to a simpler content processing workflow.

Is the benefit gained from this extra processing complexity worth it? What do implementations gain from incorporating this restriction? Does it just not work with non-square pixels for some reason?

**Comment by haudiobe** [comment]
(IOP 19/05/14) align with CMAF media profiles in the upcoming version 5 and check if it is required. If we have comments on the CMAF profile, we will contact MPEG on this. To be verified at the f2f.
